sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AuthorizationClientProperties.java

```diff
@@ -10,6 +10,8 @@
  */
 public class AuthorizationClientProperties {
 
+    public static final String AZURE_CLIENT_REGISTRATION_ID = "azure";
```

Comment:
Seems we already have `AZURE_CLIENT_REGISTRATION_ID` in `AADClientRegistrationRepository`.
Please use that instead.

Reply:
Sorry, I didn't notice that.